deployment_scripts/config

```diff
@@ -5,21 +5,17 @@ MONKEY_FOLDER_NAME="infection_monkey"
 MONKEY_GIT_URL="https://github.com/guardicore/monkey"
 
 # Monkey binaries
-LINUX_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/monkey-linux-32"
+LINUX_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.7/monkey-linux-32"
 LINUX_32_BINARY_NAME="monkey-linux-32"
-LINUX_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/monkey-linux-64"
+LINUX_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.7/monkey-linux-64"
 LINUX_64_BINARY_NAME="monkey-linux-64"
-WINDOWS_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/monkey-windows-32.exe"
+WINDOWS_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.7/monkey-windows-32.exe"
 WINDOWS_32_BINARY_NAME="monkey-windows-32.exe"
-WINDOWS_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/monkey-windows-64.exe"
+WINDOWS_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.7/monkey-windows-64.exe"
 WINDOWS_64_BINARY_NAME="monkey-windows-64.exe"
 
 # Other binaries for monkey
-TRACEROUTE_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/traceroute64"
-TRACEROUTE_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/traceroute32"
-SAMBACRY_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/sc_monkey_runner64.so"
-SAMBACRY_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.6/sc_monkey_runner32.so"
-
-# Mongo url's
-MONGO_DEBIAN_URL="https://downloads.mongodb.org/linux/mongodb-linux-x86_64-debian81-latest.tgz"
-MONGO_UBUNTU_URL="https://downloads.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1604-latest.tgz"
+TRACEROUTE_64_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.7/traceroute64"
+TRACEROUTE_32_BINARY_URL="https://github.com/guardicore/monkey/releases/download/1.7/traceroute32"
```

Comment:
Doesn't work for me, update to `https://github.com/guardicore/monkey/releases/download/v1.7.0/traceroute32` and etc.